<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>548</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>859</width>
    <height>548</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>859</width>
    <height>548</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>841</width>
      <height>381</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>100</number>
    </attribute>
    <column>
     <property name="text">
      <string>Дата</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Предыдущий день</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Следющий день</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Разница дат</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Високосный год</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Номер недели</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>След. ДР</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Дней с ДР</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>380</y>
      <width>169</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QDateEdit" name="dateEdit">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Добавить дату</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QSpinBox" name="spinBox">
           <property name="wrapping">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>Удалить дату</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>160</width>
      <height>100</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Открыть файл</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>Сохранить файл</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_7">
       <property name="text">
        <string>Очистить таблицу</string>
       </property>
      </widget>
     </item>
    </layout>
    <zorder>pushButton_5</zorder>
    <zorder>pushButton</zorder>
    <zorder>pushButton_7</zorder>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>380</y>
      <width>203</width>
      <height>114</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Рассчитать</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QDateEdit" name="dateEdit_2">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_6">
         <property name="text">
          <string>Рассчитать дату рождения</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>859</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_5</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>dateEdit_2</tabstop>
  <tabstop>pushButton_6</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>spinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
